batavia/types/Tuple.js

```diff
@@ -256,7 +256,9 @@ Tuple.prototype.__mul__ = function(other) {
     if (types.isinstance(other, types.Int)) {
         var result = new Tuple()
         for (var i = 0; i < other.valueOf(); i++) {
-            result.extend(this)
+            for (var j = 0; j < this.length; j++) {
+                result.push(this[j])
+            }
```

Comment:
This looks great, and it clearly works - but I think this is a case where the problem should be fixed a little further down stream. Calling the Python `extend()` on a tuple *should* be the right answer here; but it appears extend hasn't been implemented. Rather than duplicating what will ultimately be the logic for extend, we should be adding an extend method to Tuple.